Src/Campus.Master.API/UI/src/app/registration-page/registration-page.component.ts

```diff
@@ -37,24 +37,27 @@ export class RegistrationPageComponent implements OnInit, OnDestroy {
       {
         first_name: new FormControl(null, [
           Validators.required,
-          Validators.pattern('[a-zA-Z ]*'),
+          Validators.maxLength(50),
+          Validators.pattern('[a-zA-Z]*')
         ]),
         last_name: new FormControl(null, [
           Validators.required,
-          Validators.pattern('[a-zA-Z]*'),
+          Validators.maxLength(50),
+          Validators.pattern('[a-zA-Z]*')
         ]),
         email: new FormControl(null, [Validators.required, Validators.email]),
         password: new FormControl(null, [
           Validators.required,
           Validators.minLength(8),
+          Validators.maxLength(100),
           Validators.pattern('(?=.*?[0-9])(?=.*?[A-Z]).{8,}.+')
```

Comment:
This validation case doesn't work according to the use cases. Need to be fixed.

Reply:
Fixed